src/test/resources/features/11-allergy/allergies.feature

```diff
@@ -10,9 +10,26 @@ Feature: Allergies Management
     And a user clicks No Known Allergy button
     Then the system add no known allergies into the allergies table
     And close browser
-
+  
   @run
   Scenario: Remove No known allergies
     And a user clicks Remove No Known Allergy icon
     Then the system displays unknown in the allergies table
+    And close browser
+    
+  Scenario: Edit allergies
+    And a user clicks Edit Allergy icon
+    Then the system displays the edit Allergy page
+    When a user clicks save button
+    Then the system saves the user adjustments in the allergy table
+    And close browser
+   
+  Scenario: Delete allergies
+    And a user clicks Delete Allergy icon
+    Then the system prompts the user to confirm the transaction
+    And a user deletes the allergy
```

Comment:
user should be responding to the above system prompt and then the system performs the deletion

Reply:
Have added the usecase